### ISSUE TITLE
REPL doesn't respond to `SIGKILL` while processing some input

### ISSUE THREAD

**Issue by bugarela** [informalsystems/quint#906]
I think we need to keep a thread listening for signals so users can interrupt long processes in the REPL.

**Comment by konnov** [comment]
Yes, there is #734 on that. Since nodejs execution model is single-threaded, we would have to spawn a new process. I started to work on that, but it happened to be more complex than I anticipated.
